formal-spec/candidateproperties.tex

```diff
@@ -0,0 +1,153 @@
+\section{Candidate Properties}
+
+\subsection{Global Properties}
+
+\begin{property}[Software Update Proposal]\label{prop:submissions-for-all}
+Any stakeholder will always be able to propose a software update to be voted by the stakeholders' community and this will be recorded as an on-chain immutable event on the blockchain.
+\end{property}
+
+\begin{property}[Non Approved Software Updates]\label{prop:nonapproved-sus}
+A software update that is not approved by the stakeholders' community will never be applied,
+\end{property}
+
+\begin{property}[Approved Software Updates]\label{prop:approved-sus}
+A software update that has been approved by the stakeholders' community will be eventually applied.
+\end{property}
+
+\begin{property}[Security of Updated Ledger]\label{prop:updated-ledger-security}
+Upon activation of the changes of an approved software update for the updated ledger properties of \emph{persistence} and \emph{liveness} will hold.
+\end{property}
+
```

Comment:
BTW: you might want to set the [maximum line length](https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters) to 80 columns for LaTeX files as well.